core/embed/rust/src/ui/model_tt/component/keyboard/slip39.rs

```diff
@@ -31,7 +31,7 @@ const MAX_LENGTH: usize = 8;
 
```

Comment:
`MAX_LENGTH` constant seems to be unused now.  The same applies for `MAX_LENGTH` in `bip39.rs`.

Reply:
https://github.com/trezor/trezor-firmware/pull/3951/commits/e63cb30d53b566cbd2ddaf097bb34d6232bd5368